torch/testing/_internal/common_optimizers.py

```diff
@@ -507,8 +507,6 @@ def optim_error_inputs_func_adagrad(device, dtype):
     return error_inputs
 
 
-# TODO: consider tensor LR! See multi_tensor_optimizer_configs in test_optim.py --> tensor LR should work
-# with all implementation code paths...
```

Comment:
don't delete this todo as tensor LR still doesn't work with noncapturable foreach which is not your problem, but still a problem after this PR